submodules/power_management_services_setup/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "access_host_or_ip" {
+  description = "Jump/Bastion server Public IP to reach the target/server_host ip to configure the DNS,NTP,NFS,SQUID services"
+  type        = string
+}
+
+variable "target_server_ip" {
+  description = "Target/server_host ip on which the DNS,NTP,NFS,SQUID services will be configured."
+  type        = string
+}
+
+variable "ssh_private_key" {
+  description = "SSh private key value to login to server. It will not be uploaded / stored anywhere."
```

Comment:
SSh -> ssh

Reply:
"It will not be uploaded" - is this fully accurate - is it uploaded to the bastion host at minimum however?

Reply:
@vburckhardt: ssh key is not uploaded. we use ssh agent mechanism

Reply:
Yes accurate. The key is not stored anywhere